src/lib/eZ/RichText/Resources/stylesheets/xhtml5/edit/docbook.xsl

```diff
@@ -642,12 +642,12 @@
         <!-- Nest content of Style tag in ezcontent -->
         <xsl:when test="@data-eztype='style'">
           <xsl:element name="ezcontent" namespace="http://docbook.org/ns/docbook">
-            <xsl:apply-templates/>
+            <xsl:apply-templates />
```

Comment:
This change is neither needed nor consistent with the other calls w/o attributes.